tests/read_ntfs.rs

```diff
@@ -452,3 +452,19 @@ fn ntfs_with_duplicated_ids() {
         }
     });
 }
+
+#[test]
+// Test the sanitation of duplicated objects in collections without id
+fn ntfs_with_duplicated_objects_without_id() {
```

Comment:
For this test, I was thinking you could reuse `tests/fixtures/ntfs` or `tests/fixtures/ntfs_minimal` and then do something like this at the end.

```rust
let model = transit_model::ntfs::read("tests/fixtures/ntfs_complete_with_duplicated_ids").unwrap();
test_in_tmp_dir(|output_dir| {
	transit_model::ntfs::write(&ntm, output_dir, get_test_datetime()).unwrap();
	compare_output_dir_with_expected(
    	&output_dir,
    	None,
    	"tests/fixtures/ntfs",
);
```

Reply:
Maybe not as relevant as I thought, so let's keep it like you did.